yargs.js

```diff
@@ -173,41 +174,49 @@ function Yargs (processArgs, cwd, parentRequire) {
   }
 
   self.boolean = function (keys) {
+    argsert('<array|string>', [].slice.call(arguments))
```

Comment:
That doesn't improve the performance. Passing it to slice still requires the object be created.